pkg/api/v20191231preview/openshiftcluster.go

```diff
@@ -169,7 +169,7 @@ type WorkerProfile struct {
 	// The Azure resource ID of the worker subnet (immutable).
 	SubnetID string `json:"subnetId,omitempty"`
 
-	// The number of worker VMs.  Must be between 3 and 20 (immutable).
+	// The number of worker VMs.  Must be between 3 and 50 (immutable).
```

Comment:
I think we might be better off entirely removing `Must be ...` and `(immutable)` from our API definitions.

Reply:
Including upper band value (as I did now)?

Reply:
Yes, but I think we should do *all* the API definitions in one go in this PR.